examples/30_extended/datasets_tutorial.py

```diff
@@ -108,3 +109,39 @@
     alpha=0.8,
     cmap="plasma",
 )
+
+
+############################################################################
+# Edit a created dataset
+# =================================================
+# Use test server
```

Comment:
Please elaborate why (i.e. "Use the test server to avoid editing datasets on the live server.").